openasip/data/ProGe/tb/almaif/almaif-tb.vhdl.tmpl

```diff
@@ -0,0 +1,576 @@
+-- Copyright (c) 2002-2009 Tampere University.
```

Comment:
Pls update copyright years to be more accurate.

Reply:
updated to 2016-2023